### ISSUE TITLE
Should NFT domains be included?

### ISSUE THREAD

**Issue by PBhadoo** [publicsuffix/list#1585]
There are new types of domains in town.

https://unstoppabledomains.com

These are Crypto Based and not under ICANN.

Some browsers have added native support for them, others are on the list in the future.

Should they be included in the PSL.

They are working towards traditional DNS in the future, currently runs on Blockchain DNS.

**Comment by dnsguru** [comment]
tl;dr:  **Not likely to happen unless they are ICP-3 compliant.**  
Last I checked, HNS and Unstoppable are not ICP-3 complaint. ENS is, mostly (aside from .ETH - as in not .ETH, but other ENS names that chain onto the ICP-3 namespace like .COM / .NET, ccTLDs etc ARE possible).

> These are Crypto Based and not under ICANN.

And the PSL follows ICP-3, so **"not under ICANN" = "not going to be included in PSL"**.  This stuff is all alt-root for all intents and purposes.

We have this stuff really well defined in the guidelines and wiki.

> Some browsers have added native support for them, others are on the list in the future.

This is entirely discretionary per browser on what they do or don't opt to include but the PSL shall not force anything by breaking the adherence to ICP-3 we've had in place for > a decade

> Should they be included in the PSL.

Hard No at this point as it violates the documented standards we have in place.  If ICANN/IANA, the IAB or the IETF proposed a trial akin to what they did with test.example IDN prior to launching IDN ccTLDs back around 2004-5, it would be considered, but not without that.

Beyond that, this remains something that the browsers individually are allowed to determine, but the PSL isn't the place to include them and force anything.

As a background, the PSL is a catalog that expands detail into the ICP-3 root for more elegant domain handling.  

Browsers and other parties that use it do so on a voluntary basis, making whatever additions or deletions they opt to for their own purpose.  Adding new, unexpected stuff could trigger browsers to splinter off and do their own things, and a lot of the attraction to the PSL is that it follows norms and known standards, leveraging the guard rails that have evolved over nearly 3 decades of ICANN.  Many of the block chain projects conflict with TLDs in the ICANN space (see .music in Handshake), conflict with each other (unstoppable .crypto vs HNS .crypto vs inevitable ICANN .crytpo), or may conflict with an expanding z axis of yetanother namespace.

The PSL project that is resourced by volunteers.  In previous 'Alt-root' waves, there have been requests to add stuff from alt-roots, and in fact there were some new.net TLDs that got added at one point, where there was a .travel with entries, which ended up colliding with the .travel when ICANN delegated one in 2000s.

There was much emotional dialog about who the highlander was in the choice of what subdomains in what .travel needed cookie horizons, and it was emotionally-charged disposible volunteer cycles that got drained in trying and bottle the crazy.  The benefit of aligning to the ICP-3 adherence was that it provides an opportunity for the thinly resourced PSL volunteers to focus on the support of the existing stuff and not lose cycles to being drawn into disposible labor or emotionally expensive conflict.

Additionally, there are projects within the blockchain namespace innovation that are largely profit-focused.  Some might argue that PSL inclusion might somehow assert some form of foundational endorsement - and the PSL is only here to catalog functional ICP-3 namespace.  

Scale is another issue.  Last I looked, HNS was approaching 5M names at the top level.  We also have a scaling issue with the PSL, as it kind of perpetuates the hosts.txt issue that the DNS evolved away from in the 1980s.  We try and  keep the PSL as modest as possible in file size due to download bandwidth and appreciation of application bloat where it is included into code.

While there is an appreciation of the innovation in the blockchain space, the PSL is just not the place for it.

**Comment by PBhadoo** [comment]
Got it. Clears things.
